### PR TITLE
Swiftlint

### PR DESCRIPTION
**Pull request by pablog**
Add reporter for swiftlint.

In case it finds errors, it will fail with 0%. Each warning substracts 2%. 

It can be changed in a future applying a different value depending on the violating rule.

Here's an example of this working and uplading the report: [wolmo-networking-codestats](http://codestats.wolox.com.ar/organizations/wolox/projects/wolmo-networking-ios/branches/codestats)

### CODE REVIEW

File: config/default.yml [hunk 1]
```diff
@@ -22,13 +22,23 @@ metrics:
...
+    minimum: 70
+    upload_report: true
+    report_dir: 'swiftlint-output'
     uploader_key: <%= ENV['CODE_STATS_S3_KEY'] %>
     uploader_secret: <%= ENV['CODE_STATS_S3_SECRET_KEY'] %>
     uploader_region: <%= ENV['CODE_STATS_S3_REGION'] %>
     uploader_bucket: <%= ENV['CODE_STATS_S3_BUCKET'] %>
   tailor:
-    name: 'Code Quality'
+    name: 'Tailor'
```
Comment (epintos) [inline config/default.yml]: This is the name that will appear in codestats. You don't want to have something more representative? Same for `Swiftlint`

**Review by epintos — CHANGES_REQUESTED** [review]

File: config/default.yml (hunk 1 above)
Comment (pablog) [inline config/default.yml]: I just didn't want `Tailor` and `Swiftlint` to say `Code Quality`.

I could call them: 

- `Code Quality (Tailor)`
- `Code Quality (Swiftlint)`
- `Code Coverage (Slather)`

What do you think?

Comment (epintos) [inline config/default.yml]: I think its okay!

Comment (pablog) [inline config/default.yml]: Done

**Review by epintos — APPROVED** [review]

**Pull request merged**